fastly/fixtures/managed_logging/delete.yaml

```diff
@@ -0,0 +1,45 @@
+---
+version: 1
+interactions:
+- request:
+    body: ""
```

Comment:
I noticed that this fixture doesn't specify a `Kind` in its body (which is required, as the code logic will otherwise return an `ErrMissingKind` due to `i.Kind` zero value being zero and so would match `ManagedLoggingUnset`). Yet, the fixture suggests the response was a `200 OK` so maybe this fixture was created before that logic was implemented? Either way I think the fixture might need updating.

Reply:
Actually, I also noticed `ServiceID` is missing from the request body.

Reply:
The tests for the DELETE operation also appear to suggest that Kind/ServiceID is required and they are being validated. So I definitely think the fixture is just out of date.

Reply:
Pushed a change.  Using `Post` rather than `PostForm`. We don't need Kind or ServiceID in the body.